src/components/ProjectorWindow.js

```diff
@@ -8,18 +8,36 @@ const ProjectorWindow = () => (
         QUESTION
       </h1>
     </Alert.Heading>
-    <Jumbotron fluid>
-      <Container>
+    <Container fluid>
+      <Jumbotron fluid>
         <h1>
-          Fluid jumbotron
+          This is an example multiple choice question.
         </h1>
-        <p>
-          This is a modified jumbotron that occupies the entire horizontal space of
-          its parent.
-        </p>
-      </Container>
-    </Jumbotron>
-  </Alert>
+        <img src="https://fakeimg.pl/1280x720"
+          alt={"rich content"}
+          style={{ width: "auto", minHeight: "500px", maxHeight: "500px" }} />
```

Comment:
Use single quotes as your default, make sure that this component props will grow downwards instead of to the right.
```suggestion
        <img
          src='https://fakeimg.pl/1280x720'
          alt='rich content'
          style={{ width: 'auto', minHeight: '500px', maxHeight: '500px' }}
        />
```